### ISSUE TITLE
@origin/identity and consistency checker

### ISSUE THREAD

**Issue by mikeshultz** [OriginProtocol/origin#2871]
With the additon of `@origin/identity`, it looks like the DB schema has changed.  Add support for the new schema.

@shahthepro, has the IPFS schema changed for attestations at all?

**Comment by shahthepro** [comment]
@mikeshultz Yes, I did update the attestation schema JSON a number of times while adding new attestations.

You can check the [history of that file](https://github.com/OriginProtocol/origin/commits/master/packages/validator/src/schemas/attestation_1.0.0.json)
